round4/team-3.md

```diff
@@ -0,0 +1,117 @@
+- As a trader, I want the extension to calculate my profit and loss (PnL) for each trade, so that I can accurately evaluate the success of my trading strategies.
+- As a crypto investor, I want to easily generate tax reports for my crypto transactions so that I can comply with regulations without hassle.
+- As an avid crypto trader, I want alerts on the values of my trades so that I can make informed decisions about my crypto investments.
+- As a beginner trader, I want tips and suggestions on when to buy and sell so that I can keep learning to become a better trader.
+- As a crypto trader, I want a tool/platform that aggregates my trades across various (unique) exchanges.
+- As a portfolio manager, I want a one stop shop to see all my transactions so that I can quickly manage them all at once. 
+- As a cryptocurrency educator, I want to provide clear reports on transactions so that I can engage my students and help them develop a deeper understanding of blockchain technology and trading strategies.
+---
+
+## Notes
+Chelsea, Marc, and Rayyan don’t have much background knowledge with crypto, so they will need to do more research on the topic. However, they have a general understanding of most of the technical work.
+
+---
+
+## References & Inspiration
+https://metamask.io/
+
+---
+
+## Technical Details
+
+### User Interface
+When first downloading our extension, it will lead to our website to make the user login with WalletConnect to get their public address user id. This user id will give us access to all their public trade data.
+
+Currently, we are planning to have 2 tabs within the chrome extension. The home tab will display a graph showing the value of your trades over time. The activity tab will display all the past trades of the user in a table. That would include what type of crypto they bought, the amount bought, the value it was bought at in USD, the date bought, and the current value of the crypto. Within the extension, we want to include a refresh button to fetch latest trade data and current market prices of crypto. 
+
+If we’re able to get to it, we want to add a notification feature that alerts users when prices of their crypto increase. 
+
+### API, Libraries, and Frameworks
+- React Frontend
+- Vite 
+- Component libraries?
+- MySQL + Node.js + Express? Backend
+- WalletConnect API 
+- Have user log in to their account and get their user id from the log in. Use this info to get trade transactions
+- https://etherscan.io/
+- Viewing wallet transaction history
+- API Available: https://etherscan.io/apis
+- (Would require our own backend / server to restrict usage)
+- https://min-api.cryptocompare.com/
+- API to get current prices of crypto currencies 
+- Coingecko free (30 calls / min)
+
+
+### Data Storage
+- Storing user’s account information and etherscan transactions.
+- AWS RDS?
```

Comment:
Need Money for AWS RDS

Reply:
we're trying to avoid using a backend for now. We'll put off the AWS RDS unless we absolutely have to